mysql_scheme.sql

```diff
@@ -9,14 +8,14 @@ USE shopping;
 CREATE TABLE shopping_card (
   id              BIGINT AUTO_INCREMENT PRIMARY KEY  NOT NULL,
   username        VARCHAR(45)  NOT NULL,
-  createDate      DATE NOT NULL
+  create_date      DATE NOT NULL
 );
 
 CREATE TABLE shopping_card_item (
   id              BIGINT AUTO_INCREMENT PRIMARY KEY  NOT NULL,
-  shoppingCardId  BIGINT NOT NULL,
-  bookId          BIGINT NOT NULL,
+  shopping_card_id  BIGINT NOT NULL,
+  book_id          BIGINT NOT NULL,
   quantity        BIGINT NOT NULL,
-  createDate      DATE NOT NULL,
+  create_date      DATE NOT NULL,
   FOREIGN KEY (shoppingCardId) REFERENCES shopping_card(id)
```

Comment:
Tutaj też trzeba zmienić
```suggestion
  FOREIGN KEY (shopping_card_id) REFERENCES shopping_card(id)
```